scripts/build/docker/run-command.sh

```diff
@@ -89,6 +89,7 @@ done
 # The `-t` and TERM setup is needed to display coloured output.
 docker run -t \
   --env "TERM=xterm-256color" \
+  --env "TARGET_ARCH=$ARGV_ARCHITECTURE" \
```

Comment:
> I guess setting the TARGET_ARCH env-var is just as effective as setting the Docker entrypoint though.

What do you mean by entry point?

> Have you checked that the 32-bit files that this generates do indeed run in a pure 32-bit VM? (not just a 32-bit container using a 64-bit kernel).

Yeah, I have a Debian Jessie 32-bit installation that runs it fine.

> Although thinking about it a bit more, if we're supplying the $TARGET_ARCH anyway, is there any benefit to using separate containers, rather than just running everything in a single 64-bit container? This mixing of arches and containers and builds gets a bit confusing... ;-)

The Jessie x86 container is indeed an x64, but its customized to contain all 32-bit packages in user land, so its not quite the same.